packages/grafana-ui/src/components/Menu/Menu.tsx

```diff
@@ -43,7 +51,10 @@ export const Menu = React.forwardRef<HTMLDivElement, MenuProps>(
       </div>
     );
   }
-);
+) as MenuType;
```

Comment:
Could not find a way to type Menu with two static component props without an as here.  Works great here https://github.com/grafana/grafana/blob/main/public/app/core/components/Page/Page.tsx#L19  where we are not using 
a React.forwardRef component, but when using React.forwardRef typescript is complaining that the returned type does not have the two static props. 

Cannot figure out a way to do this other than some Object assign trick or some typescript voodoo :shrug:

Reply:
managed to figure this out using Object.assign